### ISSUE TITLE
accessibility: labels missing

### ISSUE THREAD

**Issue by libecap** [cieraclaire/animal-rescue-website#8]
Hey Ciera, the site looks great. According to the accessibility testing site you are missing <label> tags for each input field of your form
